build/antrea/ovs/apply-patches.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
since you 've already use 2.14.0, I think later it will be 2.14.2 or higher, do you still need this file?